tests/system/feature_store/test_feature_store.py

```diff
@@ -0,0 +1,354 @@
+import os
+import random
+import string
+
+import pandas as pd
+import pytest
+from storey import MapClass
+
+import mlrun
+import mlrun.feature_store as fs
+from mlrun.data_types.data_types import ValueType
+from mlrun.datastore.sources import CSVSource
+from mlrun.datastore.targets import CSVTarget, TargetTypes
+from mlrun.feature_store import Entity, FeatureSet
+from mlrun.feature_store.steps import FeaturesetValidator
+from mlrun.features import MinMaxValidator
+from tests.system.base import TestMLRunSystem
+
+from .data_sample import quotes, stocks, trades
+
+
+class MyMap(MapClass):
+    def __init__(self, multiplier=1, **kwargs):
+        super().__init__(**kwargs)
+        self._multiplier = multiplier
+
+    def do(self, event):
+        event["xx"] = event["bid"] * self._multiplier
+        event["zz"] = 9
+        return event
+
+
+def _generate_random_name():
+    random_name = "".join([random.choice(string.ascii_letters) for i in range(10)])
+    return random_name
+
+
+# Marked as enterprise because of v3io mount and pipelines
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
```

Comment:
Do you really can run the tests only on enterprise ?
I don't see you're using v3io anywhere

Reply:
all the tests read and/or write to v3io (its the default storage backend, no need to specify it)